ARCs/arc-0200.md

```diff
@@ -0,0 +1,232 @@
+      "desc": "Transfers tokens",
+      "readonly": false,
+      "args": [
+        {
+          "type": "address",
+          "name": "to_",
+          "desc": "The destination of the transfer"
+        },
+        {
+          "type": "uint256",
+          "name": "value_",
+          "desc": "Amount of tokens to transfer"
+        }
+      ],
+      "returns": { "type": "bool", "desc": "Success" }
+    },
+    {
+      "name": "transferFrom",
+      "desc": "Transfers tokens from source to destination as approved spender",
+      "readonly": false,
+      "args": [
+        {
+          "type": "address",
+          "name": "from_",
+          "desc": "The source  of the transfer"
+        },
+        {
+          "type": "address",
+          "name": "to_",
+          "desc": "The destination of the transfer"
+        },
+        {
+          "type": "uint256",
+          "name": "value_",
+          "desc": "Amount of tokens to transfer"
+        }
+      ],
+      "returns": { "type": "bool", "desc": "Success" }
+    },
+    {
+      "name": "approve",
+      "desc": "Approve spender for a token",
+      "readonly": false,
+      "args": [
+        { "type": "address", "name": "spender_" },
+        { "type": "uint256", "name": "value_" }
+      ],
+      "returns": { "type": "bool", "desc": "Success" }
+    },
+    {
+      "name": "allowance",
+      "desc": "Returns the current allowance of the spender of the tokens of the owner",
+      "readonly": true,
+      "args": [
+        { "type": "address", "name": "owner_" },
+        { "type": "address", "name": "spender_" }
+      ],
+      "returns": { "type": "uint256", "desc": "The remaining allowance" }
+    }
+  ],
+  "events": [
+    {
+      "name": "Transfer",
+      "desc": "Transfer of tokens",
+      "args": [
+        {
+          "type": "address",
+          "name": "from_",
+          "desc": "The source of transfer of tokens"
+        },
+        {
+          "type": "address",
+          "name": "to_",
+          "desc": "The destination of transfer of tokens"
+        },
+        {
+          "type": "uint256",
+          "name": "value_",
+          "desc": "The amount of tokens transferred"
+        }
+      ]
+    },
+    {
+      "name": "Approval",
+      "desc": "Approval of tokens",
+      "args": [
+        {
+          "type": "address",
+          "name": "owner_",
+          "desc": "The owner of the tokens"
+        },
+        {
+          "type": "address",
+          "name": "spender_",
+          "desc": "The approved spender of tokens"
+        },
+        {
+          "type": "uint256",
+          "name": "value_",
+          "desc": "The amount of tokens approve"
+        }
+      ]
+    }
+  ]
+}
+```
+
+Ownership of a token by a zero address indicates that a token is out of circulation indefinitely, or otherwise burned or destroyed.
+
+The methods `transfer` and `transferFrom` method MUST error when the balance of `from_` is insufficient.
```

Comment:
`transfer` has no `from_` argument. (I get your point, but the language needs to be changed.)

Reply:
Here `from` is implied as the signer of the transaction

Reply:
See also
https://github.com/algorandfoundation/ARCs/pull/223/commits/a9174bc215e756fb12858db4c0657cc0b73d2527